### PR TITLE
Add entry for SVGShapeMarkers plugin to plugin page

### PR DESCRIPTION
**Pull request by rowanwins**
Adding Leaflet.SvgShapeMarkers to plugin page

### CODE REVIEW

File: plugins.md [hunk 1]
```diff
@@ -902,6 +902,15 @@ These plugins provide new markers or news ways of converting abstract data into
 			<a href="https://github.com/iatkin">Ilya Atkin</a>
 		</td>
 	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/rowanwins/Leaflet.SvgShapeMarkers">Leaflet.SvgShapeMarkers</a>
+		</td><td>
+			Additional SVG marker types for leaflet.js such as triangle, diamond and square. Check out the <a href="http://rowanwins.github.io/Leaflet.SvgShapeMarkers/example/">demo</a>. This plugin extends the path class so you can pass any valid through any valid options such as fillOpacity or color.
```
Comment (perliedman) [inline plugins.md]: "This plugin extends the path class so you can pass any valid through any valid options such as fillOpacity or color."

There seems to be extra/missing words in this sentence, or at least I don't understand it.

Comment (rowanwins) [inline plugins.md]: Hi @perliedman 

Thanks for the feedback. 

Im saying that for example on a triangle marker you can specify the fillColor or opacity etc, anything which the [Leaflet Path class](http://leafletjs.com/reference-1.0.0.html#path) supports. Does that make sense? Happy to suggestions on how to better explain it....

Cheers

Comment (Jmuccigr) [inline plugins.md]: There's a redundant "any valid", so it should read (with some tiny additions in bold): 

so **that** you can pass ~~any valid~~ through any valid options**,** such as

**Comment by perliedman** [comment]
@rowanwins Unfortunarely you need to rebase this PR so it merges cleanly!

**Comment by rowanwins** [comment]
Urgh sorry @perliedman Im still not very good at this, Im going to close this request and start a new one which is done right :)

**Pull request closed without merging**